what-is-chalk.mdx

```diff
@@ -18,215 +18,440 @@ import { PyDiffEditor } from '@/components/Editor'
+
+Chalk evaluates features with point-in-time lookups guaranteeing evaluation only with data that would have been seen in the past, this means that you can provide labels with different past timestamps to easily get historical features that represent what your application would have retrieved online at those past times.
 
 <TemporalCorrectness />
+
+With Chalk, Data scientists easily integrate new data sources, prototype with guaranteed point-in-time correctness, and collaborate with branches and data exports.
+
+## Chalk for Data Engineers
+
+Manage features declaratively--with a few lines of code--all without the complexity of orchestrating ETL jobs, feature stores, and offline/online datastores.
+
+- cache expensive computations with configurable staleness
+- manage versioning and A/B testing of feature computation
+- create complex joins and relationships between data entities
+- configure time-window aggregations with flexible materialization options
+
+```python
+@features
+class User:
+    id: int
+    domain: str
+
+    # composite keys that can be used as join keys
+    workspace_id: str = _.domain + "-" + _.id
+    expensive_api_call: str = feature(max_staleness="30d") # cache values
+
+    # maintain different resolvers to A/B test function calls e.g. gemini vs openai
+    llm_response: str = feature(version=3)
+
+    # multi-attribute joins
+    txns: DataFrame[Transaction] = has_many(
+        lamda: (User.id == Transaction.user_id) & (User.domain == Transaction.domain)
+    )
+
+    count_txns: Windowed[int] = windowed(
+        "1d", "365d",
+        materialization={
+            # 1-day buckets for the 365d window
+            bucket_duration="1d",
+            bucket_durations={
+                # 10-minute buckets for the 1d window
+                "10m": "1d",
+            }
+        },
+        expression=_.txns.count(),
+    )
+```
+
+Chalk also integrates natively into your existing data infrastructure by deploying directly into your virtual private cloud (VPC) providing seamless resource access while maintaining strict security and compliance controls with full data isolation:
+
+    - customizable compute layer enabling the use of different memory stores (Redis, etc.) tailored to access patterns and performance requirements
+    - inherit existing security groups, policies, and ACLs
+    - co-located resources and full-control over data residency to meet compliance requirements
+
+With Chalk, Data Engineers get high-throughput, low-latency pipelines that are highly available without the overhead of configuring YAML, writing custom scripts, or stitching together disparate tools.
+
+## A full-stack solution for building and deploying enterprise AI
+
+Chalk gives AI, ML, and data teams the building blocks to prototype and deploy production AI and ML systems quickly and reliably.
+Whether you’re building a recommendation system, enterprise RAG, or fighting fraud, Chalk is the platform for low-latency, real-time ML at scale.
```

Comment:
enterprise rag doesn't really fit with the other two items here: recsys and fraud are solutions, and rag is more of a product